julearn/api.py

```diff
@@ -267,7 +274,7 @@ def run_cross_validation(  # noqa: C901
 
         if has_target_transformer:
             if isinstance(pipeline, BaseSearchCV):
-                last_step = pipeline.estimator[-1]
+                last_step = pipeline.estimator[-1]  # pyright: ignore
```

Comment:
Maybe better to have `# type: ignore` as it'll be respected by other static type checkers as well?